courses-app/src/components/Courses/components/SearchBar/SearchBar.jsx

```diff
@@ -1,20 +1,30 @@
 import "./SearchBar.css";
 import Button from "../../../../common/Button/Button";
-import Courses from "../../Courses";
+// import Courses from "../../Courses";
 
 const SearchBar = ({
   setCoursesList,
   coursesList,
   setIsCreateOpened,
   isCreateOpened,
+  setFilteredList,
 }) => {
   const addCourse = (course) => {
     setCoursesList([...coursesList, course]);
   };
   const handleSearch = () => {
     const value = document.getElementById("searchInput").value;
```

Comment:
Try to use useState for storing the search value. It will be more React based approach using useState